src/main/scala/com/netaporter/precanned/HttpServerMock.scala

```diff
@@ -19,7 +19,9 @@ object HttpServerMock {
     val empty = PrecannedResponse(HttpResponse(), Duration.Zero)
   }
 
-  case class ExpectAndRespondWith(expects: Expect, respondWith: PrecannedResponse)
+  case class ExpectAndRespondWith(expects: Expect, respondWith: PrecannedResponse, times: Option[Int] = None) {
```

Comment:
It's better to be not `times` but `numberOfTimes`. Also please specify what happens if it's None.